src/app/caesar-portal/common/services/update-group.service.ts

```diff
@@ -0,0 +1,155 @@
+///<reference path="../../../../../node_modules/@angular/http/src/http.d.ts"/>
+import { Injectable } from '@angular/core';
+import { Http, Response, Headers } from '@angular/http';
+import { Observable } from 'rxjs/Rx';
+import { Location } from '../models/location';
+import { environment } from '../../../../environments/environment';
+import { Direction } from '../models/direction';
+import { BudgetOwner } from '../models/budgetOwner';
+import { Stage } from '../models/stage';
+
+@Injectable()
+export class UpdateGroupService {
```

Comment:
usually there is no need to create additional service for update - you can keep all CRUD operations in a group service